core/domain/stats_services.py

```diff
@@ -77,7 +91,12 @@ def get_versions_for_exploration_stats(exploration_id):
 def get_exploration_stats(exploration_id, exploration_version):
     """Returns a dict with state statistics for the given exploration id.
 
-    Note that exploration_version should be a string.
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: str. The version of the exploration.
```

Comment:
StatisticsTab.js contains a js function "refershExplorationStatistics" which constructs and send the GET request to server side with 'all' as the version for exploration.
`var _EXPLORATION_STATS_VERSION_ALL = 'all';`
` $scope.refreshExplorationStatistics(_EXPLORATION_STATS_VERSION_ALL);` 
I double checked it by logging the exploration_version in the server side, as expected I am getting 'all'.